sormas-ui/src/main/java/de/symeda/sormas/ui/utils/CommitDiscardWrapperComponent.java

```diff
@@ -496,7 +498,11 @@ public void removeCommitListener(CommitListener listener) {
 	private void onCommit() {
 
 		for (CommitListener listener : commitListeners)
-			listener.onCommit();
+			try {
+				listener.onCommit();
+			} catch (CannotProceedException e) {
+				break;
```

Comment:
This exception is not thrown due to some wrong behaviour of SORMAS, but because users have created a state that requires handling. As this is not really something needing attention of someone reading the logs, I decided against generating a log entry here. Okay?

Reply:
Got it